components/navbar/sidebar.snip.html

```diff
@@ -25,7 +25,7 @@
     <ul class="list-reset m0 p0 ampstart-label">
       {{#nav}}
         {{#link}}
-          <li class="ampstart-nav-item"><a class="ampstart-nav-link" href="{{{url}}}">{{text}}</a></li>
+          <li class="ampstart-nav-item"><a class="ampstart-nav-link" href="{{{url}}}">{{{text}}}</a></li>
```

Comment:
Request to allow HTML to be passed to the text property. Our design requires the nav button letters to be evenly spaced. Unfortunately this sort of effect is not possible with CSS, so spans are required to separate the characters.